set_env_vars-sample.sh

```diff
@@ -19,4 +19,8 @@ export DYNAMO_DB_URL='http://dynamodb:8000'
 # Salesforce
 export SALESFORCE_AUTH_URL='https://test.salesforce.com'
 export SALESFORCE_CLIENT_ID=''
-export SALESFORCE_CLIENT_SECRET=''
```

Comment:
We should add these vars for test env vars?

Reply:
Added to test env vars